flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java

```diff
@@ -148,7 +148,7 @@ private static void syncDatabase(
 
     private static Map<String, String> getConfigMap(MultipleParameterTool params, String key) {
         if (!params.has(key)) {
-            return null;
+            return new HashMap<>();
```

Comment:
Is there any problem with returning null?

Reply:
Here is the initialization of tableConfig  https://github.com/apache/doris-flink-connector/blob/master/flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java#L65

Reply:
However the initialization value was reset to `null` on below:
https://github.com/apache/doris-flink-connector/blob/d49eda9c0f766b9ee2561da850a3a137c5ad7c5a/flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/CdcTools.java#L134
You can just run `CdcTools` to reproduce this without `--table-conf` parameter.

Reply:
Yes, it's possible